EventFilter/CTPPSRawToDigi/interface/DiamondVFATFrame.h

```diff
@@ -28,14 +31,14 @@ class DiamondVFATFrame : public VFATFrame
     /// get timing infromation
     uint32_t getLeadingEdgeTime() const
     {
-      uint32_t time = ((data[5]&0x1f)<<16)+data[6];
+      uint32_t time = ((data[7]&0x1f)<<16)+data[8];
```

Comment:
Indeed, this leading-trailing edges inversion was also present in 2016 data: the two following screenshots show the time over threshold &equiv; trailing - leading edge (ns) respectively before and after the patch for an arbitrary number of events in run 284029 from 2016H era for a loose selection: ![screenshot from 2017-07-18 16-41-24](https://user-images.githubusercontent.com/330969/28323084-1a9c1df4-6bd8-11e7-87da-647743d44b79.png)
![screenshot from 2017-07-18 16-33-11](https://user-images.githubusercontent.com/330969/28323081-14ffeaba-6bd8-11e7-9116-bee6aedcb3c4.png)